IntroToCSharp/Pages/Activities/Booleans.razor

```diff
@@ -0,0 +1,111 @@
+@inherits CaptainCoder.Page
+@page "/activities/booleans"
+
+
+<h1>Booleans</h1>
+<Section Title="What are booleans?">
+    <p>
+        In C# a boolean or a “bool”, is a value that attempts to answer one of the simplest questions in C#.
+        Is this statement true or false? It is important to remember that booleans are working and used to answer that question!
+        In order to create a boolean we simple make a boolean value and, depending on the situation, set it to either true or false.
+    </p>
+    <p>
+        While this is a very simple data type it can be monumentally helpful when deciding how a program should proceed, 
+        make decisions or even use logic to solve a problem. Consider this problem below.
+    </p>
+
+    <MudContainer Align="align-center">
+    <MudIcon Icon="@(FlashlightSwitch ? Icons.Material.Filled.FlashlightOn : Icons.Material.Filled.FlashlightOff)" Title="@(FlashlightSwitch ? "Flashlight On" : "Flashlight Off")"/>
+    <MudSwitch @bind-Checked="@FlashlightSwitch" ThumbIcon="@(FlashlightSwitch==true ? Icons.Material.Filled.Done : Icons.Material.Filled.Close)" ThumbIconColor="@(FlashlightSwitch==true ? Color.Success : Color.Error)">IsLightSwitchOn: @FlashlightSwitch</MudSwitch>
+    <CodeBlock Filename="Booleans/IsLightSwitchOn.cs"/>
+    </MudContainer>
+
+    <p>
+        In this problem we can see that we have a boolean called IsLightSwitchOn. This is the question that our boolean is going to try to answer. For starters, you can see that right now the light switch is set to False. In other words the question “Is the Light Switch On?” is false, 
+        meaning that the Light Switch is Off. Go ahead now and copy the code. Try turning the light switch on and then off using the boolean.
+    </p>
+</Section>
+
+<Questions Title="Exploring Booleans">
+    <ChildContent>
+        <Question Text="What would you combine a boolean with to make it more complex?" />
+        <Question Text="Can you think of another question that could be a boolean?" />
+        <Question Text="How many answers does a boolean question have?" />
+    </ChildContent>
+</Questions>
+
+<Section Title="Exploring Booleans">
+<p>
+    	We can see from this that booleans can be a lot of fun to play with by themselves but where they really shine is when you start combining 
+        them with other pieces of code! Booleans thrive when working in together with other pieces of code such as <Keyword Word="conditional operators"/> and <Keyword Word="If-Else Statements"/>, 
+        along with a whole lot of other tools at our disposal.
+</p>
+</Section>
+
+<Questions Title="Testing your knowledge!">
+    <ChildContent>
+        <p>Are these Questions Booleans?</p>
+        <p>Replace me with multiple choice quiz</p>
```

Comment:
I assume from this comment that it needs to be replaced with the new quiz component?